build.gradle

```diff
@@ -16,16 +19,15 @@ dependencies {
     compile localGroovy()
     compile 'com.google.guava:guava:18.0'
     compile 'org.eclipse.jgit:org.eclipse.jgit:4.0.2.201509141540-r'
-    compile "org.gradle:gradle-tooling-api:${gradleVersion}"
 
-    testCompile gradleTestKit()
-    testCompile ('com.blackbaud:gradle-internal-test:4.+') {
+    sharedTestCompile gradleTestKit()
+    sharedTestCompile ("com.blackbaud:gradle-internal-test:${gradleInternalMajorVersion}") {
         exclude group: 'org.spockframework'
     }
-    testCompile('org.spockframework:spock-core:1.0-groovy-2.4') {
+    sharedTestCompile('org.spockframework:spock-core:1.2-groovy-2.4') {
```

Comment:
why dont you add the dependencies plugin here?

Reply:
done